### ISSUE TITLE
Store the css inside the board's folder instead of inside every thread folder

### ISSUE THREAD

**Issue by HASJ** [bibanon/BASC-Archiver#30]
![not okay](https://cloud.githubusercontent.com/assets/9345660/12193979/73d6dbfa-b5d3-11e5-9b7e-df3d88de6fbf.jpg)
I don't think many changes are needed (talking out of my ass).
Just change the directory of the css files on the html to \board\css instead of thread\css, that way all threads can use just those files instead of writing new pointless files.

**Comment by DanielOaks** [comment]
Yeah, I've been thinking about this. The issue with this change is it means you can't just pickup a thread's folder, zip it up and give it to someone else.

We could make it a command-line and/or configuration file option though, to save it in the board folder instead of the thread folder.

Another issue is that because it's tied to the current 4chan html, when 4chan changes their layout or CSS this sort of a solution would break all the previously/newly archived threads, which is why I'm also a bit nervous to make this sort of a change.

I can see this sort of thing working really nicely with the templating solution when that gets implemented though.
